data/lists/discordz.xyz.json

```
@@ -0,0 +1,47 @@
+{
+    "id": "discordz.xyz",
+    "added": 1639382829351,
```

```suggestion
    "added": 1639382829,
```

This timestamp looks to be in milliseconds -- it is an unix timestamp, so should be in seconds.